deploy/compose/.env

```diff
@@ -14,6 +14,9 @@ COLLECTOR_OSV_API_PORT=8181
 COLLECTOR_SNYK_API_PORT=8182
 GUAC_IMAGE=ghcr.io/trustification/guac:v0.3.29
 #GUAC_IMAGE=local-organic-guac
-
+VEXINATION_WALKER_REPORT_PATH=/tmp/share/reports/vexination
```

Comment:
I think we should simplify these report settings. Ideally, the report would be by default stored at `DEFAULT_PATH+lower(REPORT_TYPE)`. This can be overriden if needed, but we wouldn't need any configuration by default.